jetson/nav/utilities.cpp

```diff
@@ -3,165 +3,135 @@
 #include <cmath>
 
 // Coverts the input degree (and optional minute) to radians.
-double degreeToRadian( const double degree, const double minute )
-{
-    return ( PI / 180 ) * ( degree + minute / 60 );
+double degreeToRadian(const double degree, const double minute) {
+    return (PI / 180) * (degree + minute / 60);
 } // degreeToRadian
 
 // Converts the input radians to degrees.
-double radianToDegree( const double radian )
-{
+double radianToDegree(const double radian) {
```

Comment:
You removed the const for this in the header file

Reply:
https://stackoverflow.com/questions/51353648/const-qualification-of-parameters-in-function-declaration

Reply:
seems like you'd want it in the header since thats the interface specification for other programmers though...